src/Twilio/Http/Request.cs

```diff
@@ -121,6 +125,9 @@ public Request(HttpMethod method, string url)
             QueryParams = queryParams ?? new List<KeyValuePair<string, string>>();
             PostParams = postParams ?? new List<KeyValuePair<string, string>>();
```

Comment:
create a new constructor.
Within request, either postparams will come or body.

Reply:
Cannot add new constructor due to conflict in other files